### ISSUE TITLE
Another Instance of: RuntimeError: bpy.types.AddonPreferences subtype could not be generated, this is a bug!

### ISSUE THREAD

**Issue by bloodychrome** [armory3d/armory#411]
Like donalffons back in October, I'm also getting this error when attempting to turn on the Armory addon (installed via itch.io, latest version V0.1) except I'm on Mac OS Sierra version 10.12.6

Here's a screenshot of the error:

![screen shot 1](https://user-images.githubusercontent.com/26043427/33792440-480b3272-dc97-11e7-94a0-e24f69e4c7fb.png)

Since donalffons' solution was to delete the userpref file, I thought I'd disable third party addons one by one to see where the conflict was. It seems as though the addon Speedflow (version 0.1.3) is the culprit, since after successfully enabling Armory, this was the only one to throw out the following error:

![screen shot 2](https://user-images.githubusercontent.com/26043427/33792449-9ff19a58-dc97-11e7-9e93-f4941d27c80f.png)

So basically at this moment I can have one or the other active at once. Not sure what the course of action is from here but I thought I'd let you know. 

On another note, any chance I have in my busy schedule I spend with Armory, it's wonderful. Thank you for doing great work!

*(optional - drag & drop zipped .blend file here)*

**Comment by luboslenco** [comment]
I tried installing and enabling Speedflow (0.1.3) but did not run into issues so far. Not sure if the issue somewhat resolved itself or I am just unable to reproduce it.

(PS: really happy you like armory when you get to play with it!)
